a03_stubs.py

```diff
@@ -0,0 +1,47 @@
+# TODO Do not edit this file directly! Instead, create a new file called
```

Comment:
We'll need to fix this again. Now that I'm viewing it in the PR review, I see we rolled back too far and created an old file (a03_stubs.py was converted to hw03_stubs.py). 

We went too far back!

Reply:
I will stay after class again tomorrow and we can look at it.